packages/pigeon/lib/cpp_generator.dart

```diff
@@ -442,10 +459,10 @@ void _writeHostApiHeader(Indent indent, Api api, Root root) {
         }
       }
       indent.addln('');
-      indent.writeln('/** The codec used by ${api.name}. */');
+      indent.writeln('/// The codec used by ${api.name}.');
```

Comment:
Here and in several places below you are using `///` directly instead of the constant.